clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -32,13 +46,27 @@
                               json-api-clients (map client-transform all-clients)]
                           {:data json-api-clients}))
 
+(defresource create-client
+             :available-media-types ["application/json"]
+             :allowed-methods [:post]
+             :handle-created (fn [context]
+                               (let [body (extract-request-body context)
```

Comment:
I think that 'let' is how you define local variables. They are local to the
scope in which they are defined.